src/test/java/ca/on/oicr/gsi/dimsum/CaseLoaderTest.java

```diff
@@ -242,6 +250,491 @@ public void testLoad() throws Exception {
+    assertEquals(1, projectsBySummary.get("PRO1").getCounts().get("FULL_DEPTH_PEND_WORK"));
+
+    // PRO2
+    assertEquals(1, projectsBySummary.get("PRO2").getCounts().get("LIBRARY_PREP_PEND_QC"));
+    assertEquals(1, projectsBySummary.get("PRO2").getCounts().get("LIBRARY_QUAL_PEND_WORK"));
+    assertEquals(1, projectsBySummary.get("PRO2").getCounts().get("LIBRARY_QUAL_PEND_QC"));
+    assertEquals(2, projectsBySummary.get("PRO2").getCounts().get("FULL_DEPTH_PEND_QC"));
+
+    // PRO3
+    assertEquals(1, projectsBySummary.get("PRO3").getCounts().get("INFORMATICS_PEND_WORK"));
+
+    // PRO4
+    assertEquals(1, projectsBySummary.get("PRO4").getCounts().get("DRAFT_REPORT_PEND_WORK"));
+    assertEquals(1, projectsBySummary.get("PRO4").getCounts().get("INFORMATICS_COMPLETE"));
+
+    // PRO5
+    assertEquals(2, projectsBySummary.get("PRO5").getCounts().get("FINAL_REPORT_PEND_WORK"));
+    assertEquals(2, projectsBySummary.get("PRO5").getCounts().get("DRAFT_REPORT_COMPLETE"));
+    assertEquals(2, projectsBySummary.get("PRO5").getCounts().get("INFORMATICS_COMPLETE"));
+
+    // PRO7
+    assertEquals(1, projectsBySummary.get("PRO7").getCounts().get("EXTRACTION_PEND_WORK"));
+
+    // PRO8
+    assertEquals(1, projectsBySummary.get("PRO8").getCounts().get("EXTRACTION_PEND_QC"));
+
+    // PRO9
+    assertEquals(1, projectsBySummary.get("PRO9").getCounts().get("LIBRARY_PREP_PEND_WORK"));
+
+    // PRO10
+    assertEquals(1, projectsBySummary.get("PRO10").getCounts().get("LIBRARY_PREP_PEND_QC"));
+
+    // PRO11
+    assertEquals(1, projectsBySummary.get("PRO11").getCounts().get("LIBRARY_QUAL_PEND_WORK"));
+
+    // PRO12
+    assertEquals(1, projectsBySummary.get("PRO12").getCounts().get("LIBRARY_QUAL_PEND_WORK"));
+
+    // PRO13
+    assertEquals(1, projectsBySummary.get("PRO13").getCounts().get("LIBRARY_QUAL_PEND_QC"));
+
+    // PRO14
+    assertEquals(1, projectsBySummary.get("PRO14").getCounts().get("LIBRARY_QUAL_PEND_QC"));
+
+    // PRO15
+    assertEquals(1, projectsBySummary.get("PRO15").getCounts().get("LIBRARY_QUAL_PEND_QC"));
+
+    // PRO16
+    assertEquals(1, projectsBySummary.get("PRO16").getCounts().get("FULL_DEPTH_PEND_WORK"));
+
+    // PRO17
+    assertEquals(1, projectsBySummary.get("PRO17").getCounts().get("FULL_DEPTH_PEND_WORK"));
+
+    // PRO18
+    assertEquals(1, projectsBySummary.get("PRO18").getCounts().get("FULL_DEPTH_PEND_QC"));
+
+    // PRO19
+    assertEquals(1, projectsBySummary.get("PRO19").getCounts().get("FULL_DEPTH_PEND_QC"));
+
+    // PRO20
+    assertEquals(1, projectsBySummary.get("PRO20").getCounts().get("LIBRARY_PREP_PEND_WORK"));
+
+    // PRO21
+    assertEquals(1, projectsBySummary.get("PRO21").getCounts().get("LIBRARY_PREP_COMPLETE"));
+
+    // PRO22
+    assertEquals(1, projectsBySummary.get("PRO22").getCounts().get("TOTAL_TESTS"));
+    assertEquals(1, projectsBySummary.get("PRO22").getCounts().get("RECEIPT_PEND_QC"));
+
+  }
+
+
+  private static Case makeCase0() {
```

Comment:
Rather than duplicating these from `CaseFilterTest`, move them to a separate `MockCases` class that you can call from both of these test classes